scijava-ops-image/src/test/java/org/scijava/ops/image/coloc/saca/SACATest.java

```diff
@@ -0,0 +1,123 @@
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ * #L%
+ */
+
+package org.scijava.ops.image.coloc.saca;
+
+import io.scif.img.ImgOpener;
+import net.imglib2.RandomAccess;
+import net.imglib2.img.Img;
+import net.imglib2.type.logic.BitType;
+import net.imglib2.type.numeric.real.FloatType;
+import net.imglib2.type.numeric.real.DoubleType;
+import net.imglib2.type.numeric.integer.UnsignedByteType;
+
+import org.scijava.io.location.FileLocation;
+import org.scijava.types.Nil;
+import org.scijava.ops.image.AbstractColocalisationTest;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/**
+ * Tests {@code coloc.saca} ops.
+ *
+ * @author Edward Evans
+ */
+
+public class SACATest extends AbstractColocalisationTest {
+
+	// XY positions to sample
+	private final static int[] xPositions = { 30, 79, 77, 104, 7, 52, 164, 88,
+		119, 65 };
+	private final static int[] yPositions = { 30, 36, 80, 79, 139, 102, 77, 41,
+		142, 118 };
+
+	// green and red colocalization images
+	private Img<UnsignedByteType> green = getPositiveCorrelationImageCh1();
+	private Img<UnsignedByteType> red = getPositiveCorrelationImageCh2();
+
+	@Test
+	public void testSACAHeatmapZScore() {
+		final double[] zscoreExpected = { 0.0, 6.117364936585281, 0.0,
+			-1.282447034877343, 0.0, 6.642396454955293, 0.0, -1.6567255788972388, 0.0,
+			3.5385003044434877 };
+
+		// get slice 15 from colocalization data
+		var gs = ops.op("transform.hyperSliceView").input(green, 2, 15).apply();
+		var rs = ops.op("transform.hyperSliceView").input(red, 2, 15).apply();
+
+		// create image container
+		Img<DoubleType> zscore = ops.op("create.img").input(gs, new DoubleType())
+			.outType(new Nil<Img<DoubleType>>()
+			{}).apply();
+
+		// run SACA heatmap Z score op
+		ops.op("coloc.saca.heatmapZScore").input(gs, rs).output(zscore).compute();
+
+		// get random access and assert results are equal
+		final RandomAccess<DoubleType> zRA = zscore.randomAccess();
+		for (int i = 0; i < xPositions.length; i++) {
+			zRA.setPosition(xPositions[i], 0);
+			zRA.setPosition(yPositions[i], 1);
+			assertEquals(zscoreExpected[i], zRA.get().getRealDouble());
+		}
+	}
+
+	@Test
+	public void testSACASigMask() {
+		final double[] sigExpected = { 0.0, 1.0, 0.0, 0.0, 0.0, 1.0, 0.0, 0.0, 0.0,
+			1.0 };
+
+		Img<FloatType> zscore_ref = openFloatImg("stats/zscore_test_data.tif");
+
+		// create image container
+		Img<BitType> sigMask = ops.op("create.img").input(zscore_ref, new BitType())
+			.outType(new Nil<Img<BitType>>()
+			{}).apply();
+
+		// run SACA significant pixel mask op
+		ops.op("coloc.saca.sigMask").input(zscore_ref).output(sigMask).compute();
+
+		// get random access and assert results are equal
+		final RandomAccess<BitType> sRA = sigMask.randomAccess();
+		for (int i = 0; i < xPositions.length; i++) {
```

Comment:
This would be awesome to use, but unfortunately for me I'm extending `AbstractColocalisationTest` and not `AbstractOpTest` where `areCongruent()` lives. I'm just going to stick with this manual way for now :D.